config/puma.rb

```diff
@@ -0,0 +1,21 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+workers Integer(ENV['WEB_CONCURRENCY'] || 1)
```

Comment:
Yes, I'm using Ruby 2.2.1 as provided by Heroku. Which reminds me to update to 2.2.2.